runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenPlugin.java

```diff
@@ -96,7 +96,11 @@ protected void shutDown() throws Exception
 		}
 
 		keyManager.unregisterKeyListener(this);
-		clientThread.invoke(this::restoreLoginScreen);
+		clientThread.invoke(() ->
+			{
```

Comment:
This formatting looks wrong

Reply:
Ah whoops, thought I'd matched the formatting of other invokes.